astropy/timeseries/tests/test_downsample.py

```diff
@@ -12,8 +12,6 @@
 
 INPUT_TIME = Time(['2016-03-22T12:30:31', '2016-03-22T12:30:32',
                    '2016-03-22T12:30:33', '2016-03-22T12:30:34'])
-ts = TimeSeries(time=INPUT_TIME, data=[[1, 2, 3, 4]], names=['a'])
```

Comment:
Good catch. Now see another benefit of doing setup inside `setup_class`, etc.

Reply:
I had to basically monkeypatch erfa to force an exception and find this. In this case `setup_class` is not strictly needed since it was just one function, but I have been paying attention to your style of testing...